robotiq_force_torque_sensor/src/rq_sensor_state.cpp

```diff
@@ -154,14 +154,27 @@ static void rq_state_start_stream()
  */
 static void rq_state_run()
 {
+	static unsigned char retries = 0;
+
 	rq_com_listen_stream();
 
 	if(rq_com_get_valid_stream() == false)
 	{
+		retries++;
+		printf("invalid stream : retry #%d\n", retries);
+
+		if (retries >= 100)
```

Comment:
ROS-I solves this by using logging macros [here](https://github.com/ros-industrial/industrial_core/blob/indigo-devel/simple_message/include/simple_message/log_wrapper.h).  By defining a compiler time macro [here](https://github.com/ros-industrial/industrial_core/blob/indigo-devel/simple_message/CMakeLists.txt#L15), the ROS or non-ROS logger versions are used.